packages/mobile/views/onboarding/views/network-setup/views/ChooseNetworkView.svelte

```diff
@@ -39,30 +56,23 @@
 
     onMount(async () => {
         if (!$onboardingProfile?.id) {
-            await initialiseOnboardingProfile(
-                $onboardingProfile?.isDeveloperProfile ?? shouldBeDeveloperProfile(),
-                NetworkProtocol.Shimmer
-            )
+            await initialiseOnboardingProfile($onboardingProfile?.isDeveloperProfile ?? shouldBeDeveloperProfile())
         }
-        updateOnboardingProfile({ networkType: null })
+        updateOnboardingProfile({ network: undefined })
```

Comment:
I think we'd also want to set `clientOptions` to undefined here too right?

Reply:
We use null for now so not planning to change it if client options is going to be moved into network